unit/src/main/include/config/configs.h

```diff
@@ -45,6 +45,46 @@ class Configs {
             cv::aruco::Board::create(kBoardArucoPts, kArucoDictionary,
                                      kArucoIds);
     };
+    class CameraMovePositions {
+      public:
+        static inline const std::vector<double> corner1Vec = {
+            Configs::Physical::kPlatformDim.width * 1.5,
+            Configs::Physical::kPlatformDim.width * 1.5,
+            Configs::Physical::kPlatformDim.width * 1.0};
+
+        static inline const std::vector<double> corner2Vec = {
+            -Configs::Physical::kPlatformDim.width * 0.5,
+            -Configs::Physical::kPlatformDim.width * 0.5,
+            Configs::Physical::kPlatformDim.width * 1.0};
+
+        static inline const std::vector<double> corner3Vec = {
+            -Configs::Physical::kPlatformDim.width * 0.5,
+            Configs::Physical::kPlatformDim.width * 1.5,
+            Configs::Physical::kPlatformDim.width * 1.0};
+
+        static inline const std::vector<double> corner4Vec = {
+            Configs::Physical::kPlatformDim.width * 1.5,
+            -Configs::Physical::kPlatformDim.width * 0.5,
+            Configs::Physical::kPlatformDim.width * 1.0};
+
+        static inline const std::vector<double> side1Vec = {
+            Configs::Physical::kPlatformDim.width * 0.5, 0,
+            Configs::Physical::kPlatformDim.width * 1.0};
+
+        static inline const std::vector<double> side2Vec = {
+            0, Configs::Physical::kPlatformDim.width * 0.5,
+            Configs::Physical::kPlatformDim.width * 1.0};
+
+        static inline const std::vector<double> side3Vec = {
+            Configs::Physical::kPlatformDim.width * 0.5,
+            Configs::Physical::kPlatformDim.width * 1.5,
+            Configs::Physical::kPlatformDim.width * 1.0};
+
+        static inline const std::vector<double> side4Vec = {
+            Configs::Physical::kPlatformDim.width * 1.5,
+            Configs::Physical::kPlatformDim.width * 0.5,
+            Configs::Physical::kPlatformDim.width * 1.0};
+    };
```

Comment:
CameraMovePositions should be a commands enum
They can be named like TOP_LEFT instead of cornerxVec
another thing you could do is to make the measurements in terms if kPlatformDim.width and just have {0.5,1.5,1.0}